cmd/juju/charmhub/convert.go

```diff
@@ -40,12 +40,14 @@ func convertCharmInfoResult(info transport.InfoResponse, arch, series string) (I
 		ir.Charm, isKubernetes = convertCharm(info)
 	}
 
+	seriesSet := set.NewStrings()
 	for _, base := range info.DefaultRelease.Revision.Bases {
 		s, _ := coreseries.VersionSeries(base.Channel)
 		if s != "" {
-			ir.Series = append(ir.Series, s)
+			seriesSet.Add(s)
 		}
 	}
+	ir.Series = seriesSet.SortedValues()
```

Comment:
Not sure if this is necessary. It might be nice to keep the original order as this tends to be chronological order